### PR TITLE
Change default chunk size from 4Kb to 64Kb

### LINKED ISSUE #1100: Make FileResponse.chunk_size variable
`FileResponse.chunk_size` is currently hardcoded to 4096. I was testing file transfer and noticed rather slow speeds compared to using scp to get the file. I increased the chunk_size and saw the speeds I was expecting. This should be configurable from `FileResponse.__init__`.

### PR DESCRIPTION
**Pull request by Kludex**
- Closes https://github.com/encode/starlette/issues/1100

@tomchristie On that issue, you said that "httpx uses 64k read size", but I couldn't find that. Do you mind pointing me out what did you mean?

### CODE REVIEW

**Comment by dalf** [comment]
The related issue in httpcore : https://github.com/encode/httpcore/issues/135
In the source code: https://github.com/encode/httpcore/blob/4f6d793e440ffeda31d7d064b434ad8083d42634/httpcore/_async/http11.py#L38

File: starlette/responses.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ async def wrap(func: typing.Callable[[], typing.Coroutine]) -> None:
 
 
 class FileResponse(Response):
-    chunk_size = 4096
+    chunk_size = 65536
```
Comment (Kludex) [inline starlette/responses.py]:
```suggestion
    chunk_size = 64 * 1024
```

**Comment by Kludex** [comment]
Thanks @dalf ! Much appreciated! :tada:

**Review by florimondmanca — APPROVED** [review]
Looking good! That's quite a similar change to the one we made in HTTPX indeed, and for the same reason (improving transfer speed by using larger chunks).

**Comment by Kludex** [comment]
Thanks @florimondmanca ! 🎉

**Pull request merged**